src/rgw/Makefile.am

```diff
@@ -92,6 +91,10 @@ librgw_la_SOURCES = \
 	rgw/rgw_xml_enc.cc \
 	rgw/rgw_website.cc
 
+if WITH_OPENLDAP
+librgw_la_SOURCES += rgw/rgw_ldap
```

Comment:
@yehudasa er, yeah

(it works as is because rgw_ldap.cc has no text--but I left it in place for future)

repushed